main.js

```diff
@@ -379,40 +377,51 @@ function addWidgetsFromJSON(jsonObject, numberOfColumns, numberOfRows) {
         const row = Math.floor(i / numberOfColumns) + 1;
         const key = column + "_" + row;
         const widget = widgetsDictionary[key];
-        if (!widget || existingWidgets.indexOf(widget) >= 0) {
-            //no widget found for this part of the grid, add an empty cell instead
-            if (!widget && existingWidgets.indexOf(widget) < 0) {
-                const id = addCell();
-                makeHiddenCell(document.getElementById(id));
-                // console.log("HIDDEN KEY : " + key);
+
+        if (widget == undefined || jsonObject.config[widget.config]) {
```

Comment:
I think you want a not in front of jsonObject here.

Reply:
🤦‍♂️